templates/blank/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.8-alpine as base
+FROM node:22.2.0-alpine as base
```

Comment:
I'd recommend to keep 20 (lts) - ie: 20.14

Reply:
@belgattitude would you mind elaborating on this? I don't mind changing it, just curious as to why?

Reply:
Long Term Support releases are generally preferred for their stability. Node 20 is, node 22 not yet. There might also have some dependencies that aren’t compatible or tested with node 22.x

Reply:
Thanks @belgattitude, I've changed it to 20.14